otherlibs/unix/mmap.c

```diff
@@ -39,8 +39,7 @@
 #endif
 
 /* Defined in [mmap_ba.c] */
-CAMLextern value
-caml_unix_mapped_alloc(int flags, int num_dims, void * data, intnat * dim);
+value caml_unix_mapped_alloc(int, int, void *, intnat *);
```

Comment:
I would prefer `extern value ...`.  The function is defined in another file, even though it's a file from the same library.  

For function declarations, `extern` and no modifier have the same effect (not even sure), but not for variable declarations, so let's be disciplined and put `extern` when it's defined in another file.

Reply:
I don't have the standard to reference it, but my understanding is that `extern` is implicit on function declarations, but it doesn't matter - we can of course choose to have it everywhere! Change pushed.